README.md

```diff
@@ -245,7 +245,13 @@ List of IDs of namespaces of which all pages should be excluded.
 
 Default value: `[ 2, 4, 8, 12 ]` (excluding `User`, `Project`, `MediaWiki` and `Help`)
 
-Example: `$wgPageNetworkExcludedNamespaces = [ NS_USER, NS_PROJECT ];`
+Examples: 
+
+Exclude the User and Project namespaces:
+`$wgPageNetworkExcludedNamespaces = [ NS_USER, NS_PROJECT ];`
+
+Keep the default exclusions and also exclude the Talk namespace:
+Example: `$wgPageNetworkExcludedNamespaces[] = [ NS_TALK ];`
```

Comment:
I believe that should be:

`$wgPageNetworkExcludedNamespaces[] = NS_TALK;`

Reply:
You are absolutely right. I think I hopefully fixed it now. I was so focused on not understanding the git process that I missed the boat on what I was actually doing. I don't know if I need to do anything else with this conversation to close it off.